velox/expression/fuzzer/ExpressionFuzzer.cpp

```diff
@@ -902,18 +896,31 @@ core::TypedExprPtr ExpressionFuzzer::generateArgFunction(const TypePtr& arg) {
     }
   }
 
+  core::TypedExprPtr body;
   if (eligible.empty()) {
-    return std::make_shared<core::LambdaTypedExpr>(
-        ROW(std::move(names), std::move(args)),
-        generateArgConstant(returnType));
+    body = generateArgConstant(returnType);
+    if (options_.referenceQueryRunner == nullptr ||
+        options_.referenceQueryRunner->isSupported(body)) {
+      return std::make_shared<core::LambdaTypedExpr>(
+          ROW(std::move(names), std::move(args)), body);
+    } else {
+      return std::make_shared<core::LambdaTypedExpr>(
+          ROW(std::move(names), std::move(args)),
+          generateArgColumn(returnType));
```

Comment:
Good question. I don't think this cause a big change to the lambda expression coverage though. This change is basically saying making the lambda to be something like `(x, ...) -> c0` instead of the original `(x, ...) -> timestamp '1970-01-01 00:00:00'` when the ConstantTypedExpr is not supported. So for other supported ConstantTypedExpr, the lambda still returns the constant literal. Given that lambdas returning a constant literal are already trivial themselves, I don't think making the lambda returning a column instead would hurt the coverage.

The comment definitely should be updated.